manifests/configmap.yaml

```diff
@@ -11,9 +11,10 @@ data:
 
   debug_logging: "true"
   workers: "4"
-  docker_image: registry.opensource.zalan.do/acid/demospilo-10:1.3-p3
+  docker_image: registry.opensource.zalan.do/acid/spilo-10:1.4-p6
+  pod_service_account_name: "zalando-postgres-operator"
   secret_name_template: '{username}.{cluster}.credentials'
-  # etcd_host: ""
+  etcd_host: ""
```

Comment:
This is done only to overwrite [the default hostname](https://github.com/zalando-incubator/postgres-operator/blob/987b43456b080518c23a1242caa5c199ae7c1407/pkg/util/config/config.go#L72) that does not resolve on `minikube`.

What is our current policy on using/configuring etcd ?

Reply:
Correct me if I'm wrong; as far as I know:

* etcd is still supported as a configuration alternative for Operator-managed Spilo clusters, and currently there are no plans to stop supporting it.

* etcd is definitely not the recommended configuration — the recommended configuration is using the Kubernetes API as the DCS backing Operator-managed Spilo clusters, which requires setting `etcd_host` to the empty string in the operator `ConfigMap`.

* I would argue `etcd_host` should default to the empty string.

Reply:
Yes, ETCD remains supported by Patroni but is not our suggested way to run on Kubernetes. This would just be another service to be maintained that we can avoid for the "native" experience.